views/chitter_home.erb

```diff
@@ -0,0 +1,34 @@
+<!-- GET / -->
+
+<html>
+  <head><head>
+  <body>
+    <h1>Hello! Welcome to Chitter!</h1>
+
+    <form method="POST" action="/new_member"> 
+      <label>Not a member?  </label>
+      <input type="submit" name="Sign up!"><a href="/new_member"></a>" /> <!-- (link) new_member -->
```

Comment:
Add a HREF to the local host to connect for example href="https://www.codewars.com/users/IAmNotAbbas">CodeWars</a>